exercises/armstrong-numbers/.meta/template.j2

```diff
@@ -0,0 +1,13 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        input = {{ case["input"]["number"] }}
+        expected = {{ case["expected"] }}
+        self.assertEqual({{ case["property"] | to_snake }}(input), expected)
```

Comment:
I would suggest putting everything in the `assertEqual` call:
```
self.assertEqual(
    {{ case["property"] |to_snake }}({{ case ["input"]["number"] }}), 
    {{ case["expected"] }}
)
```

The generated result will produce a smaller diff from the original.

Reply:
Done :)